network/load-balancer/concepts.mdx

```diff
@@ -44,7 +52,23 @@ For more information about balancing rules, refer to our [blog post "What is a l
 
 </Concept>
 
-<Concept opened>
+<Concept>
+
+## Certificate
+
+An certificate (aka SSL certificate / TLS certificate / TLS/SSL certificate) is a digital certificate that enables an encrypted connection between a client and a web server. 
+
+<Message type="tip">
+
+  You may hear certificates referred to as "SSL certificates", "TLS certificates" or "TLS/SSL certificates". These are all the same thing. SSL was the protocol initially used for encryption, though modern certificates in fact use TLS (Transport Layer Security) as the encryption protocol.
```

Comment:
Be careful, it's not the certificate that uses an encryption protocol. The certificate is used by the encryption protocol to verify the server is legit. Here I would say "SSL (Secured Socket Layer) was the protocol initially used for encryption, though it has now been replaced with TLS (Transport Layer Security) as the encryption protocol.